src/main/cpp/blaze.cc

```diff
@@ -821,7 +822,8 @@ static void ConnectOrDie(const OptionProcessor &option_processor,
     }
   }
   BAZEL_DIE(blaze_exit_code::INTERNAL_ERROR)
-      << "couldn't connect to server (" << server_pid << ") after 120 seconds.";
+      << "couldn't connect to server (" << server_pid << ") after "
+      << startup_options.local_startup_timeout_secs << " seconds.";
```

Comment:
why not use try_until_time?

Reply:
Ack. Lost this in my inbox.

Hm, I don't understand the suggestion. `try_until_time` is a point in time, but we're displaying an error to the user indicating the timeout _duration_. Am I missing something?

Reply:
I have no idea now what I was thinking. Sorry!